### DISCUSSION TITLE
Build fail during Hugging Face Spaces Gradio App set up

### DISCUSSION THREAD

**Discussion by BCBeeravelly** [mrdbourke/pytorch-deep-learning#543] [Q&A]
I am trying to build the app, however, there has been an error which I have tried to resolve but could not arrive at any solution. Please find the discussion and the build log in the below link. Please make sure to check my commits too as I have tried making changes in requirements.txt too. Thanks in advance!
[My build log](https://huggingface.co/spaces/bbeeravelly/food-vision-mini/discussions/1#64b265a569b9fea014a8ba5b)

**Comment by ds-kenwatanabe** [comment]
See the errors: 
ERROR: Could not find a version that satisfies the requirement torch==2.0.1+cu117 (from versions: 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1)
ERROR: No matching distribution found for torch==2.0.1+cu117

Try removing the "+cu117" from the requirements.txt file and you should be good.
